x/programs/examples/simulator/server/server.go

```diff
@@ -0,0 +1,236 @@
+package main
```

Comment:
don't review this file yet, probably wont have time to review it before friday's demo. Much of the stuff is specifically for the demo, many things will likely change before then.

Reply:
"io/ioutil" is deprecated :)